src/service/ratelimit.go

```diff
@@ -225,7 +225,7 @@ func NewService(runtime loader.IFace, cache limiter.RateLimitCache,
 		for {
 			logger.Debugf("waiting for runtime update")
 			<-newService.runtimeUpdateEvent
-			logger.Debugf("got runtime update and reloading config")
+			logger.Warnf("got runtime update and reloading config")
```

Comment:
I think info would be better?

Reply:
Info would work too, used Warn level because that was the default log level in settings.go and there are almost no Info level logs in the code base.

Reply:
Yeah I think that is historical, though I think it's a mistake (of my own making). WDYT about changing the default to info and changing the various warns that should be info to info?

Reply:
I like that. Would feel much more natural that way.